config/settings/pen.yml

```diff
@@ -11,7 +11,7 @@ dfe_sign_in:
   profile: https://profile.signin.education.gov.uk
 
 recruits_api:
-  base_url: "https://sandbox.apply-for-teacher-training.service.gov.uk/register-api"
+  base_url: "https://pen.apply-for-teacher-training.service.gov.uk/register-api"
```

Comment:
Is there a `pen` environment for the Apply service (`apply-for-teacher-training.service.gov.uk`)?

Reply:
Hi Tom, there is no pentesting environment for apply-for-teaching , I will fix this by reverting .

Reply:
fixed